src/components/menuBar/menu-bar.spec.js

```diff
@@ -26,11 +26,27 @@ describe('material.components.menuBar', function() {
       });
 
       describe('ARIA', function() {
+        
         it('sets role="menubar" on the menubar', function() {
           var menuBar = setup();
           var ariaRole = menuBar[0].getAttribute('role');
           expect(ariaRole).toBe('menubar');
         });
+        
+        it('should set the role on the menu trigger correctly', inject(function($compile, $rootScope) {
```

Comment:
Does this test throw the error if you don't fix the code? If so, I think it's good, but can you update the strings to have the spaces inside the string so that we can use code formatting tools? 😄